packages/docusaurus-init/templates/bootstrap/package.json

```diff
@@ -19,7 +19,8 @@
     "@mdx-js/react": "^1.5.8",
     "classnames": "^2.2.6",
     "react": "^17.0.1",
-    "react-dom": "^17.0.1"
+    "react-dom": "^17.0.1",
```

Comment:
theme bootstrap is unmaintained currently, we'll make it production ready later but it's not worth to update it atm